var/spack/repos/builtin/packages/libristra/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
I think the style check is failing because of this:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Reply:
Fixed. Is there something other than `spack style` we can call to catch those?